test/integration2/query/insert/inserts.spec.js

```diff
@@ -2315,6 +2315,72 @@ describe('Inserts', function () {
               assertJsonEquals(result[0].content, arrayOfObject);
             });
         });
+
+        describe(`#5769 sqlite multi-insert uses standard syntax and supports JSON (does NOT include better-sqlite3)`, function () {
+          it(`should JSON.stringify Sqlite multi-insert of Objects`, async function () {
+            if (!isSQLite3(knex)) {
```

Comment:
do neweer better-sqlite support this feature?

Reply:
I'll update the test and check. I think so, when I was originally authoring this I was just trying to focus on making sure that `sqlite3` worked.